docs/events/api/timeline-api-update.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[timeline-api-update]]
+=== Add a note to existing timeline
+
+Add a note to existing timeline or an event of existing timeline
+
+==== Request URL
+
+`PATCH <kibana host>:<port>/api/note`
+
+==== Request body
+
+A JSON object defining the note content to be added.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+|`note` | <<note-object-schema, Note>> a|`noteSavedObject` to be added:
```

Comment:
Hi @angorayc `<<note-object-schema>>` isn't a valid reference either because this ID tag doesn't exist anywhere. Let me know which section it's supposed to reference and I can add the ID there.

Reply:
Should be the content of `eventNotes-obj`